decidim-admin/app/views/decidim/admin/participatory_process_user_roles/_table.html.erb

```diff
@@ -0,0 +1,34 @@
+<section id="process_admins">
+  <h4><%= t(".process_admins_title", scope: "decidim.admin") %></h4>
+
+  <%= form_for :participatory_process_user_role, url: participatory_process_user_roles_path(@participatory_process) do |f| %>
+    <table class="stack">
+      <thead>
+        <tr>
+          <th><%= t("models.participatory_process_user_role.fields.email", scope: "decidim.admin") %></th>
```

Comment:
You can use Rails' `human_attribute_name` for this: https://github.com/rails/rails/blob/eebb9ddf9ba559a510975c486fe59a4edc9da97d/activemodel/lib/active_model/translation.rb#L43

Note that it's using `I18n` under the hood.

I'm aware we're not doing it this way in other parts of the app, but maybe we should start to.